config.json

```diff
@@ -1923,6 +1923,14 @@
         "practices": ["lists", "classes", "for-loops"],
         "prerequisites" :["lists"],
         "difficulty": 2
+      },
+      {
+        "slug": "square-root",
+        "name": "Square Root",
+        "uuid": "61886554-ec84-422a-bbf9-aeee37c45bb6",
+        "practices": ["numbers"],
```

Comment:
One thing that bugs me is that we only have the concepts `for-loops` and `foreach-loops`, and here students probably need a `while` loop, which technically isn't covered by these concepts.

Reply:
That's unfortunate. Definitely something to look into later.